browser/src/slideshow/SlideShowPresenter.ts

```diff
@@ -146,6 +146,9 @@ class SlideShowPresenter {
 			this.centerCanvas();
 		} else {
 			this._stopFullScreen();
+			// we need to cleanup current/prev slide,
+			// Escape handler is not called as we disabled it above
+			this._slideShowNavigator.endPresentation(false);
```

Comment:
It seems to me that by passing `false` to `SlideShowNavigator.endPresentation` in the endless case we end up invoking `startTimer`: see `SlideShowPresenter.endPresentation`. Moreover that last method already takes care to invoke `_stopFullScreen`. So maybe it could be enough to invoke `SlideShowNavigator.quit()` inside the `else` branch, as it's done when the separate window is closed (see `this._windowCloseInterval = setInterval ...` in `_doInWindowPresentation`).
Anyway I didn't test it.

Reply:
quit is also fine, done